site/src/components/TableCellLink/TableCellLink.tsx

```diff
@@ -0,0 +1,42 @@
+import Link from "@material-ui/core/Link"
+import { makeStyles } from "@material-ui/core/styles"
+import TableCell, { TableCellProps } from "@material-ui/core/TableCell"
+import { Link as RouterLink } from "react-router-dom"
+import { combineClasses } from "../../util/combineClasses"
+
+// TableCellLink wraps a TableCell filling the entirety with a Link.
+// This allows table rows to be clickable with browser-behavior like ctrl+click.
+export const TableCellLink: React.FC<
+  TableCellProps & {
+    to: string
+  }
+> = (props) => {
+  const styles = useStyles()
+
+  return (
+    <TableCell className={styles.cell} {...props}>
+      <Link
+        component={RouterLink}
+        to={props.to}
+        className={combineClasses([styles.link, "MuiTableCell-root", "MuiTableCell-body"])}
+      >
+        {props.children}
+      </Link>
+    </TableCell>
+  )
+}
+
+const useStyles = makeStyles((theme) => ({
+  cell: {
+    padding: "0 !important",
```

Comment:
Why do we need `!important` in these styles?

Reply:
Unfortunately the MUI styles will override, and that can't work for the padding (because this needs to fill the entire cell).

Reply:
Got it. I wonder if we could use specificity in class naming to get around this. Not a blocker for this PR.

Reply:
I shall try!

Reply:
I shall try!